samples/StaticHtmlComposites/webpack.config.dev.js

```diff
@@ -0,0 +1,6 @@
+const config = require('./webpack.config.js');
+
+module.exports = {
+  ...config,
+  devtool: 'eval-source-map'
```

Comment:
Is it worth adding a comment about what this does. It took quite a while for us to figure out, so let's make it easy for us to remember :)